packages/expo-image-manipulator/ios/ImageManipulatorUtils.swift

```diff
@@ -0,0 +1,27 @@
+import SDWebImageWebPCoder
+import Photos
+
+/**
+ Returns pixel data representation of the image.
+ */
+func imageData(from image: UIImage, format: ImageFormat, compression: Double) -> Data? {
+  switch format {
+  case .jpeg, .jpg:
+    return image.jpegData(compressionQuality: compression)
+  case .png:
+    return image.pngData()
+  case .webp:
+    return SDImageWebPCoder.shared.encodedData(with: image, format: .webP, options: [.encodeCompressionQuality: compression])
+  }
+}
+
+/**
+ Checks if we are dealing with a ph asset URL and uses the correct method to fetch it.
+ */
+func retreiveAsset(from url: URL) -> PHAsset? {
```

Comment:
`retreiveAsset` -> `retrieveAsset`
That's how not waiting for my spellchecking review ends up 😂

Reply:
😅 Damn it